packages/components/atoms/f-form-field/package.json

```diff
@@ -55,10 +55,8 @@
     "@justeat/browserslist-config-fozzie": ">=1.1.1"
   },
   "devDependencies": {
-    "@vue/cli-plugin-babel": "4.5.16",
-    "@vue/cli-plugin-unit-jest": "4.5.16",
-    "@vue/test-utils": "1.0.3",
     "@justeat/f-wdio-utils": "1.x",
-    "postcss-assets": "5.0.0"
+    "postcss-assets": "5.0.0",
+    "@vue/cli-plugin-babel": "4.5.16"
```

Comment:
Had to leave this dependency as it was causing issues when moved to the root. We believe it's just an issue with optional chaining and our babel config, so we can address this as part of a later ticket if other components exhibit the same behaviour.